README.md

```diff
@@ -31,10 +31,9 @@ Including e.g. the schema id in the namespace name avoids any collision.
 
 ## Backwards compatibility
 
-Please, avoid changes which break binary compatibility.  Flatbuffers documentation contains
-good information about how to maintain binary compatibility.
-If you need to make breaking changes, acquire a new schema id.
```

Comment:
I would leave this line in, perhaps qualifying it with "unless the schema was clearly marked as _under development_".